integration_test.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
This script fails locally for me because the tests haven't been compiled. Also setting up testing might be worth a short addition to the README.md

Reply:
So the integration test passing was actually a false positive, if you look at it the script failed in CI also. I think the script itself must not be returning the exit code correctly. So maybe this is another thing to fix, making sure the integration tests fail if anything returns a non-zero code, perhaps this is as easy as adding `set -e` to the script? See the CI failure here: https://github.com/utam0k/youki/pull/37/checks?check_run_id=2673869152#step:6:18